### PR TITLE
`.py` file

### PR DESCRIPTION
**Pull request by sayakpaul**

### CODE REVIEW

File: examples/vision/handwriting_recognition.py [hunk 1]
```diff
@@ -0,0 +1,498 @@
+"""
+Title: Handwriting recognition
+Authors: [Aakash Kumar Nain](https://twitter.com/A_K_Nain), [Sayak Paul](https://twitter.com/RisingSayak)
```
Comment (AakashKumarNain) [inline examples/vision/handwriting_recognition.py:3]: `A_K_Nain`, consistent with other code examples

File: examples/vision/handwriting_recognition.py [hunk 2]
```diff
@@ -0,0 +1,498 @@
+"""
+Title: Handwriting recognition
```
Comment (AakashKumarNain) [inline examples/vision/handwriting_recognition.py:2]: Recognition*

File: examples/vision/handwriting_recognition.py [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+Keras provides different preprocessing layers to deal with different modalities of data.
+[Here](https://keras.io/guides/preprocessing_layers/) you can get a comprehensive
+introduction about all of them. Our example involves preprocessing labels at a character
+level. This means that if there are two labels namely "cat" and "dog" then our character
+vocabulary should be - (a, c, d, g, o, t) (without any special tokens). We will use the
+[`StringLookup`](https://keras.io/api/layers/preprocessing_layers/categorical/string_looku
+p/) layer for this purpose. 
+"""
+
+from tensorflow.keras.layers.experimental.preprocessing import StringLookup
```
Comment (AakashKumarNain) [inline examples/vision/handwriting_recognition.py:176]: We should group the imports at the top

File: examples/vision/handwriting_recognition.py [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+    _, ax = plt.subplots(4, 4, figsize=(15, 8))
+
+    for i in range(16):
+        img = images[i]
+        img = tf.image.flip_left_right(img)
+        img = tf.transpose(img, perm=[1, 0, 2])
+        img = (img * 255.0).numpy().clip(0, 255).astype(np.uint8)
+        img = img[:, :, 0]
+
+        # Gather indices where label!= 99.
```
Comment (AakashKumarNain) [inline examples/vision/handwriting_recognition.py:311]: != padding_token

File: examples/vision/handwriting_recognition.py [hunk 5]
```diff
@@ -0,0 +1,498 @@
...
+
+"""
+## Training
+"""
+
+epochs = 30  # To get good results this should be at least 50.
+
+# Train the model.
+model = build_model()
+history = model.fit(train_ds, validation_data=validation_ds, epochs=epochs)
```
Comment (AakashKumarNain) [inline examples/vision/handwriting_recognition.py:433]: We can add an `early stopping` callback as well

File: examples/vision/handwriting_recognition.py [hunk 6]
```diff
@@ -0,0 +1,498 @@
...
+
+        ax[i // 4, i % 4].imshow(img, cmap="gray")
+        ax[i // 4, i % 4].set_title(label)
+        ax[i // 4, i % 4].axis("off")
+
+
+plt.show()
+
+"""
+We can notice that the original image is kept as intact as possible and has been
```
Comment (AakashKumarNain) [inline examples/vision/handwriting_recognition.py:326]: We can see that the content of the original image ...

**Comment by AakashKumarNain** [comment]
Thanks @sayakpaul . Looking good. Just some minor changes and we can then do a PR

File: examples/vision/handwriting_recognition.py (hunk 2 above)
Comment (sayakpaul) [reply examples/vision/handwriting_recognition.py:2]: I ket it lower-caps on purpose as some of the examples do follow this pattern. But it looks like both of us prefer to keep titles proper-cased.

Comment (AakashKumarNain) [reply examples/vision/handwriting_recognition.py]: Haha yeah

**Comment by AakashKumarNain** [comment]
LGTM! Let's do a PR and get feedback from Francois

**Pull request merged**